ta4j-core/src/main/java/org/ta4j/core/aggregator/DurationBarAggregator.java

```diff
@@ -135,7 +135,7 @@ public List<Bar> aggregate(List<Bar> bars) {
             }
 
             if (!onlyFinalBars || i <= bars.size()) {
-                final Bar aggregatedBar = new BaseBarBuilder().timePeriod(timePeriod)
+                final Bar aggregatedBar = new BaseBarBuilder(null).timePeriod(timePeriod)
```

Comment:
Why null here? If a null `numFactory `param is a common use case when instantiating `BaseBarBuilder`, instead of passing null as part of its API usage, `BaseBarBuilder `should have a default, no param constructor which then sets the null value internally.

Reply:
done